cypress/support/component.tsx

```diff
@@ -58,24 +72,28 @@ Cypress.Commands.add('mount', (component, options, rerenderKey) => {
     );
   });
   return mount(
-    <MemoryRouter initialEntries={['/1']}>
+    <MemoryRouter initialEntries={route?.initialEntries || ['/1']}>
       <PageFramework>
         <ActiveUserProvider>
           <Page>
             <Routes>
-              <Route path="/:id" element={component} />
+              <Route path={`${route?.path || '/:id'}`} element={component} />
             </Routes>
           </Page>
         </ActiveUserProvider>
       </PageFramework>
     </MemoryRouter>,
-    options,
-    rerenderKey
+    {},
```

Comment:
This is hard to grok/parse. Can you add a comment to clarify why the mount method needs these extra parameters? Somewhat related; can these params be optional?

Reply:
good call. These are the default values there, so they should be okay to omit entirely